generators/app/index.js

```diff
@@ -159,7 +159,7 @@ module.exports = class extends Generator {
                 clearCoverage: "shx rm -rf coverage",
                 karma: "run-s clearCoverage karma-ci",
                 lint: "eslint ./**/webapp/**/*.js && prettier --check ./**/webapp/**/*.{js,xml}",
```

Comment:
Maybe we keep both in sync? 

```
                lint: "eslint ./**/webapp/**/*.js --no-error-on-unmatched-pattern && prettier --plugin-search-dir=. --check ./**/webapp/**/*.{js,xml} --no-error-on-unmatched-pattern",
```

Reply:
strange enough this error is only when with `execa.commandSync` in `basic.js`
so the error is not coming up yet, but will may come soon,
so, yeah, makes sense to change it there too just to be sure

Reply:
done